ui/api/transport.go

```diff
@@ -184,17 +185,15 @@ func decodeChannelsCreation(_ context.Context, r *http.Request) (interface{}, er
 }
 
 func decodeChannelUpdate(_ context.Context, r *http.Request) (interface{}, error) {
-	if !strings.Contains(r.Header.Get("Content-Type"), contentType) {
-		return nil, errors.ErrUnsupportedContentType
-	}
-
+	// if !strings.Contains(r.Header.Get("Content-Type"), contentType) {
+	// 	return nil, errors.ErrUnsupportedContentType
+	// }
+	fmt.Println("paspdapsdpaspdapsdpaspdpasdpaspdpasdapsdpaspdpasd")
```

Comment:
OK to keep validation commented for now, but remove these printfs as they polute the code